src/components/Jobs/utils/convertDate.js

```diff
@@ -0,0 +1,35 @@
+const INVALID_DATE = 'Invalid Date';
+
+export const DATE_TYPES = {
+  string: 'string',
+  number: 'number',
+  Date: 'Date',
+};
+
+/**
+ * Converts date type.
+ * Returns first parameter if date or type are not valid values.
+ *
+ * @param  {string | number | Date} date Date value
+ * @param  {string} [toType = 'string'] Type to convert
+ *
+ * @returns  {string | number | Date} Converted date
+ */
+export const convertDate = (date, toType = DATE_TYPES.string) => {
+  const _date = new Date(date);
```

Comment:
There is no convention to name a kind of private variables starting with `_`. Please choose more appropriate name.